src/enums/incorporatorTypes.ts

```diff
@@ -1,4 +1,4 @@
 export enum IncorporatorTypes {
     PERSON = 'person',
-    CORPORATION = 'org'
+    CORPORATION = 'organization'
```

Comment:
Would you mind renaming IncorporatorTypes.CORPORATION to IncorporatorTypes.ORGANIZATION?

Reply:
Sure. Will do it.

Reply:
Thanks!